src/app/login/login-form.component.ts

```diff
@@ -51,30 +51,30 @@ export class LoginFormComponent {
 
   createUser({ name, password, repeatPassword }: {name: string, password: string, repeatPassword: string}) {
     if (password === repeatPassword) {
-      this.couchService.put('_users/org.couchdb.user:' + name,
-      { 'name': name, 'password': password, 'roles': [], 'type': 'user', 'isUserAdmin': false })
+      this.couchService.put('_users/org.couchdb.user:' + name.toLowerCase(),
+      { 'name': name.toLowerCase(), 'password': password, 'roles': [], 'type': 'user', 'isUserAdmin': false })
         .subscribe((data) => {
-          this.message = 'User created: ' + data.id.replace('org.couchdb.user:', '');
+          this.planetMessageService.showMessage('User created: ' + data.id.replace('org.couchdb.user:', ''));
           this.welcomeNotification(data.id);
           this.login(this.model, true);
-        }, (error) => this.message = '');
+        }, (error) => this.planetMessageService.showMessage(error.error.reason));
     } else {
-      this.message = 'Passwords do not match';
+        this.planetMessageService.showMessage('Passwords do not match');
     }
   }
 
   login({ name, password }: {name: string, password: string}, isCreate: boolean) {
-    this.couchService.post('_session', { 'name': name, 'password': password }, { withCredentials: true })
+    this.couchService.post('_session', { 'name': name.toLowerCase(), 'password': password }, { withCredentials: true })
       .pipe(switchMap((data) => {
         // Post new session info to login_activity
         this.userService.set(data);
         return this.userService.newSessionLog();
       })).subscribe((res) => {
         if (isCreate) {
-          this.router.navigate( [ 'users/update/' + name ]);
+          this.router.navigate( [ 'users/update/' + name.toLowerCase() ]);
         } else {
           this.reRoute();
         }
-      }, (error) => this.message = 'Username and/or password do not match');
+      }, (error) => this.planetMessageService.showMessage(error.error.reason));

```

Comment:
Same as above, but let's pass in `Username and/or password do not match` since that will be the most common error response.